OctopusDSC/Tests/cOctopusServerActiveDirectoryAuthentication.Tests.ps1

```diff
@@ -105,6 +105,7 @@ try
 
             Context 'Set-TargetResource' {
                 BeforeAll {
+                    $dscHelpersPath = Resolve-Path "$PSCommandPath/../../OctopusDSCHelpers.ps1"
```

Comment:
This doesn't feel right. If anything, if it needs changing, the line below should reference `$script:dscHelpersPath` instead.
If it does need to change, it should be in a separate PR, because its unrelated to this change.

Reply:
Removed to PR #268